themes/default/content/blog/serverless-arch-templates/index.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: "Serverless Arch Templates"
-date: 2022-10-25
+date: 2022-11-7
```

Comment:
I think you need to use `2022-11-07` here. I noted the preview link does not show the blog post at all.